### PR TITLE
Use clap derive macros

### PR DESCRIPTION
**Pull request by Skrilltrax**

### CODE REVIEW

File: hcctl/src/main.rs [hunk 1]
```diff
@@ -18,6 +17,31 @@ struct Settings {
     ua: Option<String>,
 }
 
+/// Command-line tool for interacting with a https://healthchecks.io account
+#[derive(Clap)]
+#[clap(version = env ! ("CARGO_PKG_VERSION"))]
```
Comment (msfjarvis) [inline hcctl/src/main.rs:22]: What is this formatting o_O

File: hcctl/Cargo.toml [hunk 2]
```diff
@@ -9,6 +9,6 @@ edition = "2018"
 [dependencies]
 anyhow = "1.0.33"
 chrono = "0.4.19"
-clap = { version = "3.0.0-beta.2", default-features = false, features = ["std", "suggestions", "color"] }
+clap = "3.0.0-beta.2"
```
Comment (msfjarvis) [inline hcctl/Cargo.toml:12]: Don't do that, just turn on the `derive` feature instead.

**Review by msfjarvis — CHANGES_REQUESTED** [review]
The code in `monitor/` also needs to be migrated.

**Comment by Skrilltrax** [comment]
@msfjarvis yeah this was just a test to get initial feedback, will update monitor too.

File: hcctl/src/main.rs (hunk 1 above)
Comment (Skrilltrax) [reply hcctl/src/main.rs:22]: Idk why CLion is doing that

File: hcctl/Cargo.toml (hunk 2 above)
Comment (msfjarvis) [reply hcctl/Cargo.toml:12]: The default features adds `cargo` which has the `crate_version!()` macro so this can stay as-is

**Review by msfjarvis — APPROVED** [review]

**Pull request merged**